tests/rptest/tests/redpanda_kerberos_test.py

```diff
@@ -178,13 +178,16 @@ def _license_nag_is_set(self):
         )
 
     @cluster(num_nodes=3)
+    @ok_to_fail_fips  # See NOTE below
```

Comment:
I didn't see a way to check if fips is enabled, but since it's only the licence nag that should be ignored, this seems like a large hammer; ok_to_fail should really be a tool for temporary workarounds.

Reply:
Is there any written guidance as to when to use `@ok_to_fail`?  Do you want me to rewrite the test so if it's in FIPS mode it just exits and pass?